### PR TITLE
[cli] Handle 400 errors as user errors

### PR DESCRIPTION
**Pull request by styfle**
When a user performs an action such as trying to add a domain they don't own, the API returns a 400-499 response code.

Previously, we would save this error in Sentry but this PR changes the behavior so that the error is printed locally for the user but not saved in Sentry.

### CODE REVIEW

File: packages/now-cli/src/index.js [hunk 1]
```diff
@@ -639,6 +654,12 @@ const main = async argv_ => {
       return 1;
     }
 
+    if (err instanceof APIError && 400 <= err.status && err.status <= 499) {
+      err.message = err.serverMessage;
+      output.prettyError(err);
+      return 1;
+    }
```
Comment (styfle) [inline packages/now-cli/src/index.js:660]: This is the relevant code block for this PR, the other changes are prettier formatting only

File: packages/now-cli/src/util/response-error.ts [hunk 2]
```diff
@@ -20,12 +19,9 @@ export default async function responseError(
...
-    message = bodyError.message;
   }
 
-  if (message == null) {
-    message = fallbackMessage === null ? 'Response Error' : fallbackMessage;
-  }
+  const msg = bodyError?.message || fallbackMessage || 'Response Error';
 
-  return new APIError(message, res, bodyError);
+  return new APIError(msg, res, bodyError);
```
Comment (styfle) [inline packages/now-cli/src/util/response-error.ts:26]: I simplified the logic in this function but it should be equivalent

File: packages/now-cli/src/index.js [hunk 3]
```diff
@@ -190,8 +191,10 @@ const main = async argv_ => {
   } catch (err) {
     console.error(
       error(
-        `${'An unexpected error occurred while trying to find the ' +
-          'global directory: '}${err.message}`
+        `${
+          'An unexpected error occurred while trying to find the ' +
+          'global directory: '
+        }${err.message}`
```
Comment (TooTallNate) [inline packages/now-cli/src/index.js]:
```suggestion
        `An unexpected error occurred while trying to find the global directory: ${
          err.message
        }`
```

**Review by pacocoursey — APPROVED** [review]
LGTM

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**